app/src/main/res/layout/screen_patient_prescribed_drugs_entry.xml

```diff
@@ -23,33 +23,36 @@
     android:paddingBottom="@dimen/spacing_16" />
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    style="@style/Clinic.V2.BottomAlignedButtonFrame"
-    android:backgroundTint="@color/blue1">
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/spacing_8"
```

Comment:
We need to disable clip to padding here or else the button elevations will not be displayed.
```xml
android:clipToPadding="false"
```